srcpkgs/id3lib/patches/c99-bool.patch

```diff
@@ -0,0 +1,23 @@
+Needed for C99 bool compatability when stdbool.h is included:
+https://sourceforge.net/p/id3lib/mailman/message/30500558/
+
+diff --git a/include/id3/globals.h b/include/id3/globals.h
+index 5d8fa20..733ab3f 100644
+--- a/include/id3/globals.h
++++ b/include/id3/globals.h
+@@ -82,13 +82,13 @@
+ 
+ #define ID3_C_VAR extern
+ 
+-#ifndef __cplusplus
++#if !defined(__cplusplus) && !defined(__bool_true_false_are_defined)
+ 
+ typedef int bool;
+ #  define false (0)
+ #  define true (!false)
+ 
+-#endif /* __cplusplus */
++#endif /* __cplusplus __bool_true_false_are_defined */
```

Comment:
Hm, thinking a bit more, this look wrong.
`stdbool.h` and C++'s `bool` size could be 1 char.
But this header specifically `typedef int bool` which would be either 4 or 8,
if they don't use bool as a struct member, there're nothing wrong, however, `struct Mp3_Headerinfo` has 3 `bool` for `privatebit`, `copyrighted` and `original` squeeze together. This already break ABI between C and C++.
And with this change break ABI for C, too.

Reply:
And this need a revbump.

Reply:
I see what you mean. Is it safe to assume gcc will define C++ bool and bool from stdbool.h in a compatible way, given that we expect C++ to be able to call C code?  In that case, we could simply remove the whole block and `#include <stdbool.h>`

Reply:
Posted before I saw your changes.  That's exactly what you did.